cargo/private/cargo_utils.bzl

```diff
@@ -2,6 +2,20 @@
 
 load("//rust/platform:triple_mappings.bzl", "system_to_binary_ext")
 
+# TODO: remove after dropping support for Bazel < 7 when `abs` is a global
```

Comment:
Our minimum supported bazel version is already 7.3.1 - I'm happy to merge this if it's convenient for folks, but just FYI we don't officially support Bazel 6 :)